cloud/azure/modules/app/variables.tf

```diff
@@ -59,13 +52,9 @@ variable "bastion_address_prefixes" {
 }
 
 variable "app_sku" {
-  type        = map(string)
+  type        = string
   description = "SKU tier/size/capacity information"
-  default = {
```

Comment:
**question:** Why don't we need the tier and capacity anymore?

Reply:
@dkatzz The tier and capacity can be inferred from the sku_name. 

@jeffpw-goog mindlessly copying the old pattern which set defaults inside of variables.tf and overrode them again inside of main.tf. There isn't a good reason for doing this and I think it adds more confusion than not. I'll remove the unnecessary definitions inside of main.tf